tools/Release/CreatePullRequest.ps1

```diff
@@ -0,0 +1,48 @@
+<#
+.PARAMETER versionPath
+  The path to the msbuild props file where the version number is specified
+#>
+
+Param(
+  [string]
+  $versionPath
+)
+
+if ($versionPath -eq "")
+{
+  $versionPath = "$PSScriptRoot\..\CustomMSBuild\Versioning.props"
+}
+
+$versions = New-Object xml
+$versions.PreserveWhitespace = $true
+$versions.Load($versionPath)
+foreach ($propertyGroup in $versions.Project.PropertyGroup)
+{
+  if ($propertyGroup.VersionRelease -ne $null)
+  {
+    $versionNumber = $propertyGroup.VersionMajor.'#text' + "." + $propertyGroup.VersionMinor.'#text' + "." + $propertyGroup.VersionBuildNumber.'#text'
+
+    break;
+  }
+}
+
+if ($versionNumber -eq $null)
+{
+  Write-Error "No version number was found in $versionPath."
+  Exit
+}
+
+$branchName = "releases/$versionNumber"
+
+$branchName
+
+git checkout -b $branchName
+
+git add *
+git commit -m "revving version number to $versionNumber and updating PublicAPI.Shipped.txt files to reflect API changes for this release"
```

Comment:
revving typo?

Reply:
I actually thought this wasn't a word and was going to change it, but according to bing:
> rev
> [rev]
> VERB
> revving (present participle)
> increase the running speed of (an engine) or the engine speed of (a vehicle) by pressing the accelerator, especially while the clutch is disengaged: